src/utils/version.ts

```diff
@@ -157,7 +159,6 @@ export function getPackageVersion(): string {
  * Returns the stringified version of the passed SemVer object.
  */
 export function semVerToString(s: SemVer) {
-  return `${s.major}.${s.minor}.${s.patch}${s.pre ? `-${s.pre}` : ''}${
-    s.build ? `+${s.build}` : ''
-  }`;
+  return `${s.major}.${s.minor}.${s.patch}${s.pre ? `-${s.pre}` : ''}${s.build ? `+${s.build}` : ''
```

Comment:
craft is going to have some other problems with this -- currently we don't allow `+` releases anywhere -- is this necessary?

Reply:
this is a craft version check. I've hit this when testing the compiled craft locally - it comes with version `1.11.0-dev.0` and even though you specify that as `minVersion` in `.craft.yml`, the check fails without the changes I've made here.